packages/lyra/tests/lyra.test.ts

```diff
@@ -364,9 +364,91 @@ describe("lyra", () => {
       tolerance: 3,
     });
 
+    // this now finds 3, which I think is correct?
```

Comment:
Hi @tlahav  👋🏽
I ran the tests and now it returns duplicated results.
Are you still working on this I guess right?
<img width="827" alt="Screenshot 2022-07-25 093239" src="https://user-images.githubusercontent.com/78874117/180723729-b5786401-1982-4750-afc0-a3c3fae0c819.png">

Reply:
This definitely should not return 3 results, since there are only 2 documents in the index. @thomscoder is right about the duplicated result.

Reply:
Couldn't have worded better. Thanks @lucaong !!